README.md

```diff
@@ -15,3 +15,189 @@ To store endpoint data:
 
 - MongoDB >= 3.6 as we need dotted key support (e.g. `{ 'key.path': 'xyz' }`)
 - Your MongoDB set up needs to be a replica set
+
+## Setup Mongo Replica Set
+
+### Yarn Mapping Mediator
```

Comment:
This title reads a bit strange, Seems like Yarn developed a mapping mediator :), Maybe update it to be `Yarn setup`, and the other one can be `Docker setup`

```suggestion
### Yarn Setup
```